test/src/specs/rpc/set_ban.rs

```diff
@@ -27,6 +28,7 @@ impl Spec for RpcSetBan {
             None,
             Some("for_test".to_owned()),
         );
+        sleep(1);
```

Comment:
Better add a wait loop. I think we already have such helpers.


```
CHECK_COUNT=3
while E is not true and CHECK_COUNT > 0:
  sleep(S)
  CHECK_COUNT-=1
```

Reply:
Yes, let change it into "wait_until"